package/fusemodule/package

```diff
@@ -0,0 +1,73 @@
+e!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(fusemodule)
+pkgdesc="FUSE (Filesystem in Userspace) Kernel Module"
+archs=(rm1)
+url=https://github.com/libfuse/libfuse
+pkgver=1.0.0-1
+timestamp=2021-04-06T22:16Z
+section=utils
+maintainer="plan5 <30434574+plan5@users.noreply.github.com>"
+license=GPL-2.0-only
+makedepends=(build:bc build:lzop build:git)
+flags=(nostrip)
+
+_kernelrepo=https://github.com/remarkable/linux
+_kernelrev=1774e2a6a091fdc081324e966d3db0aa9df75c0b
+_defconfig=arch/arm/configs/zero-gravitas_defconfig
+
+image=base:v1.3.2
+source=(
+)
+sha256sums=(
+)
```

Comment:
```suggestion
source=()
sha256sums=()
```

Reply:
Or omit the two fields altogether since [they’re optional](https://github.com/toltec-dev/toltec/blob/testing/docs/package.md#source-field).

Reply:
I've removed them